tests/test_app.py

```diff
@@ -125,11 +125,6 @@ def test_configure_warnings():
     warnings.warn("A warning")
     assert "A warning" in handler.messages['warning'][0]
 
-    handler.reset()
-    warnings.warn("Pending deprecation", PendingDeprecationWarning)
-    warnings.warn("Deprecation", DeprecationWarning)
-    assert len(handler.messages['warning']) == 1
-
```

Comment:
This should be covered in the lines below

Reply:
The purpose is to test that only DeprecationWarnings are reported, and i think it fails with both warnings being reported, so I think there's an acutal problem somewhere?

Reply:
I think this might be `pytest` specific https://github.com/pytest-dev/pytest/issues/4013#issuecomment-423677736
and I added an explicit ignore for PendingDeprecationWarnings